### PR TITLE
Update exec team 2020-2021 and fix format command

### PR DESCRIPTION
**Pull request by Vassenx**
### List of changes:
- Update exec team information and images for 2020-2021
- npm run format:js command was not finding the .js/.jsx files

### Type of change:

- [x] Chore
- [x] Bug

### PR Checklist

- [x] Merged `master` branch before testing
- [x] I have linted/formatted my code locally
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,11 +97,11 @@
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:md": "remark content/posts/",
     "write-good": "write-good $(glob 'content/posts/**/*.md')",
-    "format:js": "eslint --fix '**/*.{js,jsx}'"
+    "format:js": "eslint --fix --ext .js,.jsx ."
```
Comment (loreina) [inline package.json:100]: tested format script on `master`, works fine for me. does this syntax change anything?

Comment (Vassenx) [reply package.json:100]: Before I was getting
No files matching the pattern "'**/*.{js,jsx}'" were found.

Comment (Vassenx) [reply package.json:100]: I moved finance up by prez and just wanted the external and internal next to each other as their roles are similar

Comment (Vassenx) [reply package.json:100]: Adding the old exec back into a different folder!

Comment (loreina) [reply package.json:100]: just curious, are you still getting that eslint error when you run the old script? new one works for me, just wonder where the error is coming from

Comment (Vassenx) [reply package.json:100]: Yep, tried again with the old command using npm run format:js and still got it. Looks like just a syntax issue. Another fix could be to remove the apostrophes: "format:js": "eslint --fix src/**/*.{js,jsx}"

**Pull request merged**